tests/mock/test_mock_model.py

```diff
@@ -238,6 +239,25 @@ def test_get_ilc_data(self):
             ilc_status = self.model._get_ilc_data()["status"]
             self.assertEqual(ilc_status[0], idx % 16)
 
+    def test_calculate_force_error_tangent(self):
+
+        self.model.control_open_loop.inclinometer_angle = 89.853
+        tangent_force_current = np.array(
+            [-325.307, -447.377, 1128.37, -1249.98, 458.63, 267.627]
+        )
+        data = self.model._calculate_force_error_tangent(tangent_force_current)
+
+        force_error_tangent = data["force"]
+        self.assertEqual(force_error_tangent[0], -325.307)
```

Comment:
Here and in the following assertions, you are using hard coded values. Consider extracting these into variables and unifying them in a single location you could easily update afterwards. 

Something like a method:

```
def get_expected_force_error_tangent(self):
    return ...
```

then here you could do:

```
expected_force_error_tangent, expected_force_error_tangent_tolerance = self.get_expected_force_error_tangent()

for value, expected, tolerance in zip(force_error_tangent, expected_force_error_tangent, expected_force_error_tangent_tolerance):
    if tolerance > 0.0:
        self.assertAlmostEqual(value, expected, tolerance)
    else:
        self.assertEqual(value, expected)
```

Reply:
Done in the commit: [06acb16](https://github.com/lsst-ts/ts_m2com/pull/12/commits/06acb16998adc75ae725c1c2354e208a48fad9a9). Thanks!